pkg/testutil/testutil.go

```diff
@@ -800,6 +787,9 @@ func newBase(t *testing.T, ns string, ipv6Compatible bool, kubernetesCompatible
 	} else if !base.EnableKubernetes && base.KubernetesCompatible {
 		t.Skip("runner skips Kubernetes compatible tests in the non-Kubernetes environment")
 	}
+	if !GetFlakyEnvironment() && !GetEnableKubernetes() && !GetEnableIPv6() {
```

Comment:
This is what keeps the status-quo for existing tests (and makes them still run with the retries).
As we rewrite more tests, they will join the no-retry bucket.